nekoyume/Assets/_Scripts/UI/Module/Inventory.cs

```diff
@@ -160,6 +160,16 @@ private void OnTabButtonClick(IToggleable toggleable, ItemType type, System.Acti
             }
 
             SetToggle(equipmentButton, ItemType.Equipment);
+            if (useConsumable)
+            {
+                ReactiveAvatarState.Inventory.First().Subscribe(e =>
+                    {
+                        SetInventory(e, onUpdateInventory);
+                        if (_consumables.Any()) SetToggle(consumableButton, ItemType.Consumable);
+                    })
+                    .AddTo(_disposablesOnSet);
```

Comment:
```suggestion
                    });
```
If you use `First()` in observable stream, It would be auto-disposed by first data.
So, not needs `AddTo()`.